.changeset/twelve-poets-kick.md

```diff
@@ -0,0 +1,7 @@
+---
+'@leafygreen-ui/stepper': patch
```

Comment:
I think we're marking file restructuring as `major`. @brookeyalof do you know?

Reply:
I think it's just a patch